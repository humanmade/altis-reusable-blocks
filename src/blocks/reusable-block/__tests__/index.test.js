import { name, options } from '../';

describe( 'name', () => {

	test( 'is as expected', () => {
		expect( name ).toBe( 'altis/reusable-block' );
	} );

} );

describe( 'options', () => {

	test( 'are as expected', () => {
		expect( options ).toMatchSnapshot( {
			description: expect.any( String ),
			title: expect.any( String ),
		} );
	} );

} );
