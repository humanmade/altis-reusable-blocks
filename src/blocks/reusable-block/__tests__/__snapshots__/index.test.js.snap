// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`options are as expected 1`] = `
Object {
  "category": "common",
  "description": Any<String>,
  "edit": [Function],
  "icon": "controls-repeat",
  "save": [Function],
  "supports": Object {
    "customClassName": false,
    "html": false,
    "reusable": false,
  },
  "title": Any<String>,
}
`;
